x-pack/docs/en/security/securing-communications/tls-versions-jdk.asciidoc

```diff
@@ -0,0 +1,145 @@
+enable SSL and TLS versions that would otherwise be disabled.
+
+The JDK configuration file lists the SSL/TLS algorithms that are disabled in that JDK.
+Removing a TLS version from that list will allow it to be used on the JDK.
+To do this, you should follow these steps:
+
+1. Locate the configuration file for your JDK.
+2. Copy the `jdk.tls.disabledAlgorithms` setting from that file, and add it to a custom
+   configuration file within the {es} configuration directory.
+3. In the custom configuration file, remove the value for the TLS version you want
+to use from `jdk.tls.disabledAlgorithms`.
+4. Configure {es} to pass a custom system property to the JDK so that your custom
+   configuration file is used.
+
+===== Locate the configuration file for your JDK
+
+For the {es} **bundled JDK**, the configuration file is in a sub directory of
+the {es} home directory (`$ES_HOME`):
+
+* Linux: `$ES_HOME/jdk/conf/security/java.security`
+* Windows: `$ES_HOME/jdk/conf/security/java.security`
+* macOS:`$ES_HOME/jdk.app/Contents/Home/conf/security/java.security`
+
+For **JDK8**, the configuration file is within the `jre/lib/security` directory
+of the Java installation.
+If `$JAVA_HOME` points to the home directory of the JDK that you use to run {es},
+then the configuration file will be in:
+
+* `$JAVA_HOME/jre/lib/security/java.security`
+
+For **JDK11 or later**, the configuration file is within the `conf/security`
+directory of the Java installation.
+If `$JAVA_HOME` points to the home directory of the JDK that you use to run
+{es}, then the configuration file will be in:
+
+* `$JAVA_HOME/conf/security/java.security`
+
+===== Copy the disabledAlgorithms setting
+
+Within the JDK configuration file is a line that starts with
+`jdk.tls.disabledAlgorithms=`.
+This setting controls which protocols and algorithms are _disabled_ in your JDK.
+The value of that setting will typically span multiple lines.
+
+For example, in OpenJDK 16 the setting is:
+[source,text]
+--------------------------------------------------
+jdk.tls.disabledAlgorithms=SSLv3, TLSv1, TLSv1.1, RC4, DES, MD5withRSA, \
+    DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL
+--------------------------------------------------
+
+Create a new file in your in your {es} configuration directory named `es.java.security`.
+Copy the `jdk.tls.disabledAlgorithms` setting from the JDK's default configuration file into `es.java.security`.
+You do not need to copy any other settings.
+
+===== Enable required TLS versions
+
+Edit the `es.java.security` file in your {es} configuration directory, and
+modify the `jdk.tls.disabledAlgorithms` setting so that any SSL or TLS versions
+that you wish to use are no longer listed.
+
+For example, to enable TLSv1.1 on OpenJDK 16 (which uses the
+`jdk.tls.disabledAlgorithms` settings shown above), then the contents of the
+`es.java.security` file would contain the previously disabled TLS algorithms
```

Comment:
```suggestion
For example, to enable TLSv1.1 on OpenJDK 16 (which uses the
`jdk.tls.disabledAlgorithms` settings shown previously), the
`es.java.security` file would contain the previously disabled TLS algorithms
```

Reply:
We try to avoid locational words like "above" and "below", which are not easily parsed by non-sighted users. Our docs use these words in many places, but it's best to avoid them if possible.